lib/launcher.js

```diff
@@ -217,12 +217,13 @@ class Launcher {
             this.logBuffer.add({ level: 'system', msg: 'Updating project dependencies' })
             pkg.dependencies = wantedDependencies
             fs.writeFileSync(pkgFilePath, JSON.stringify(pkg, null, 2))
+            const npmEnv = Object.assign(process.env, this.settings.env)
```

Comment:
```suggestion
            const npmEnv = Object.assign({}, process.env, this.settings.env)
```

`Object.assign`s first param is `target` mutated  (everything on the right is merged into `target`)

Add `{}` as the first param to avoid altering `process.env`